shared/shared_test.go

```diff
@@ -0,0 +1,105 @@
+package shared_test
+
+import "fmt"
+import "os"
+import "github.com/bobo333/sponge/shared"
+import "testing"
+
+func TestGetEnvVar(t *testing.T) {
+	// nonexistent should return an error
+	val, err := shared.GetEnvVar("DOES_NOT_EXIST")
```

Comment:
you can validate that this env var doesn't exist (and maybe select a different one by adding a number if it does) to ensure the test doesn't fail just because someone accidentally also used DOES_NOT_EXIST as an env var.